Source/WebKit/NetworkProcess/storage/OriginStorageManager.h

```diff
@@ -83,6 +83,7 @@ class OriginStorageManager : public CanMakeWeakPtr<OriginStorageManager> {
     void closeCacheStorageManager();
     String resolvedPath(WebsiteDataType);
     bool isActive();
+    bool hasDataInMemory();
```

Comment:
const, probably for isActive and isEmpty as well.

Reply:
They are not const because they might initialize the default StorageBucket.